content/influxdb/clustered/query-data/execute-queries/client-libraries/go.md

```diff
@@ -26,7 +26,7 @@ list_code_example: |
 
     func Query() error {
         client, err := influxdb3.New(influxdb3.ClientConfig{
-            Host:       "https://cluster-id.influxdb.io",
+            Host:       "https://cluster-host.com",
```

Comment:
Why not `"{{% influxdb/host %}}"`? I'm probably forgetting why.

Reply:
Because this is in the frontmatter. Shortcodes don't work inside of frontmatter.